src/khoj/processor/conversation/anthropic/anthropic_chat.py

```diff
@@ -149,6 +155,7 @@ def converse_anthropic(
     query_images: Optional[list[str]] = None,
     vision_available: bool = False,
     tracer: dict = {},
+    attached_files: str = None,
```

Comment:
I liked `attached_files` because it gives a connotation for the relationship between the messages and the documents in primitives analogous to how we think about messaging genrally. Query is a bit generic.